docs/architecture/overview/transaction-lifecycle.mdx

```diff
@@ -52,7 +52,7 @@ Data about the transaction and the state of the network at its time of execution
 in **traces**, an output of part of the sequencer called the [trace generator](../stack/sequencer/traces-generator.md).
 
 Traces are passed to the state manager block-by-block and then used to update the network state.
-Once state is up to date, you'll see the transaction reflected and confirmed in your wallet.
+Once state is up-to-date, you'll see the transaction reflected and confirmed in your wallet.
```

Comment:
```suggestion
Once state is up to date, you'll see the transaction reflected and confirmed in your wallet.
```

Reply:
The existing usage was correct. It should only be hyphenated when modifying a noun, which it isn't doing in this instance